charts/operator-wandb/charts/parquet/values.yaml

```diff
@@ -17,6 +17,9 @@ extraCors: []
 
 common: {}
 deployment: {}
+cronJob:
+  enabled: false
+  schedule: "11 * * * *"
```

Comment:
Note: Arbitrary number to avoid staggering of jobs at the hour.

Reply:
```
  backfillByRuns:
    enabled: false 
    schedule: "11 * * * *"
```

Reply:
`backfillByRuns` or something more relavant.

Reply:
```
cron
  backfillByRuns:
    enabled: false 
    schedule: "11 * * * *"
```

Reply:
Very wise, I agree with Yo.